<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PV Drone Inspect Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWebEngineView" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Dataset"/>
    <addaction name="actionDataset_Settings"/>
    <addaction name="actionClose_Dataset"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionNew_Analysis"/>
   </widget>
   <widget class="QMenu" name="menuAnnotation">
    <property name="title">
     <string>Annotation</string>
    </property>
    <addaction name="actionNew_Defect_Annotation"/>
    <addaction name="actionLoad_Defect_Annotation"/>
    <addaction name="actionSave_Defect_Annotation"/>
    <addaction name="actionClose_Defect_Annotation"/>
    <addaction name="separator"/>
    <addaction name="actionAnnotate_Strings"/>
    <addaction name="actionExport_String_Annotation"/>
    <addaction name="actionClose_String_Annotation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnnotation"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Dataset">
   <property name="text">
    <string>Open Dataset...</string>
   </property>
  </action>
  <action name="actionClose_Dataset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Dataset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAnnotation_Editor">
   <property name="text">
    <string>Annotation Editor</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Defect_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Defect Annotation...</string>
   </property>
  </action>
  <action name="actionSave_Defect_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Defect Annotation</string>
   </property>
  </action>
  <action name="actionLoad_Defect_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Defect Annotation...</string>
   </property>
  </action>
  <action name="actionNew_Analysis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Analysis...</string>
   </property>
  </action>
  <action name="actionAnnotate_Strings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Annotate Strings...</string>
   </property>
  </action>
  <action name="actionClose_String_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close String Annotation</string>
   </property>
  </action>
  <action name="actionSave_String_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save String Annotation</string>
   </property>
  </action>
  <action name="actionDiscard_Defect_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Discard Defect Annotation</string>
   </property>
  </action>
  <action name="actionDiscard_String_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Discard String Annotation</string>
   </property>
  </action>
  <action name="actionClose_Defect_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Defect Annotation</string>
   </property>
  </action>
  <action name="actionExport_String_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export String Annotation...</string>
   </property>
  </action>
  <action name="actionDataset_Settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dataset Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
